tests/vxlan/conftest.py

```diff
@@ -70,3 +72,5 @@ def pytest_addoption(parser):
         type=int,
         help="Highest expected src port for VXLAN UPD packet"
     )
+
+    add_wr_arp_args(parser)
```

Comment:
I suspect this is the offending line caused the issue. The `run_tests.sh` script needs to do a pretest by running command like:

```
pytest --inventory veos_vtb --host-pattern vlab-01 --testbed vms-kvm-t0 --testbed_file vtestbed.csv --log-cli-level warning --log-file-level debug --kube_master unset --showlocals --assert plain --show-capture no -rav --ignore=ptftests --ignore=acstests --
ignore=saitests --ignore=scripts --ignore=k8s --ignore=sai_qualify --junit-xml=logs/1vlan/pretest.xml --log-file=logs/1vlan/pretest.log --topology util -m pretest
```

This will try to collect all the test cases matching mark `pretest`. While collecting cases under `tests/arp`, the `add_wr_arp_args` function will be called while evaluating `tests/arp/conftest.py`. This will define the `test_duration` option.

While collecting cases under `tests/vxlan`, file `tests/vxlan/conftest.py` will be evaluated again. Then this `add_wr_arp_args` function is called for the second time and triggered the issue.

So, the best practice is not to import anything from other feature folder under `tests` folder. 
Otherwise it could trigger issue like this. Another problem is that this will cause dependencies between tests scripts for different features.